WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -0,0 +1,57 @@
+
+## Daily routine
+
+1. Start
+2. Check code review requests. [link](https://github.com/zkSNACKs/WalletWasabi/pulls) __Reviews / Awaiting review from you__
+2. Check [priority issues!](https://github.com/zkSNACKs/WalletWasabi/pulls?q=is%3Aopen+is%3Apr+label%3Apriority). Take it or set Assignees and Labels if you can.
```

Comment:
```suggestion
2. Check [priority issues!](https://github.com/zkSNACKs/WalletWasabi/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Apriority+). Take it or set Assignees and Labels if you can.
```
The proposed link by nopara contains PRs also. We can use this one instead

Reply:
I prefer to see everything that is priority.

Reply:
1- If so then you shouldn't say `Check [priority issues!]`, you should maybe say `Check [priority issues & PRs!]`.
2- The current link displays only PRs.
This is the correct link https://github.com/zkSNACKs/WalletWasabi/labels/priority